doc/rbd/rbd-exclusive-locks.rst

```diff
@@ -27,57 +27,55 @@ Exclusive locking is mostly transparent to the user.
-multiple clients, it needs to acquire the lock specifically with
-``RBD_LOCK_MODE_EXCLUSIVE``.
+.. note::
+   It is possible for two or more concurrently running processes to open the
+   image and to read from it. The client acquires the exclusive lock only when
+   attempting to write to the image. To disable transparent lock transitions
+   between multiple clients, the client must acquire the lock with
+   ``RBD_LOCK_MODE_EXCLUSIVE``.
 
 
 Blocklisting
 ============
 
-Sometimes, a client process (or, in case of a krbd client, a client
-node's kernel thread) that previously held an exclusive lock on an
-image does not terminate gracefully, but dies abruptly. This may be
-due to having received a ``KILL`` or ``ABRT`` signal, for example, or
-a hard reboot or power failure of the client node. In that case, the
-exclusive lock is never gracefully released. Thus, when a new process
-starts and attempts to use the device, it needs a way to break the
-previously held exclusive lock.
-
-However, a process (or kernel thread) may also hang, or merely lose
-network connectivity to the Ceph cluster for some amount of time. In
-that case, simply breaking the lock would be potentially catastrophic:
-the hung process or connectivity issue may resolve itself, and the old
-process may then compete with one that has started in the interim,
-accessing RBD data in an uncoordinated and destructive manner.
-
-Thus, in the event that a lock cannot be acquired in the standard
-graceful manner, the overtaking process not only breaks the lock, but
-also blocklists the previous lock holder. This is negotiated between
-the new client process and the Ceph Mon: upon receiving the blocklist
-request,
+Sometimes a client process (or, in case of a krbd client, a client node's
+kernel thread) that previously held an exclusive lock on an image does not
+terminate gracefully, but dies abruptly. This may be because the client process
+received a ``KILL`` or ``ABRT`` signal, or because the client node underwent a
+hard reboot or suffered a power failure. In cases like this, the exclusive lock
+is never gracefully released. This means that any new process that starts and
+attempts to use the device must break the previously held exclusive lock.
+
+However, a process (or kernel thread) may hang or merely lose network
+connectivity to the Ceph cluster for some amount of time. In that case,
+breaking the lock would be potentially catastrophic: the hung process or
+connectivity issue could resolve itself and the original process might then
+compete with one that started in the interim, thus accessing RBD data in an
+uncoordinated and destructive manner.
+
+In the event that a lock cannot be acquired in the standard graceful manner,
+the overtaking process not only breaks the lock but also blocklists the
+previous lock holder. This is negotiated between the new client process and the
+Ceph Mon. Upon receiving the blocklist request:
```

Comment:
Is "Mon" with this capitalization used elsewhere?  FWIW I usually spell out "monitor" since, unlike OSD and MDS, there isn't much to abbreviate.

Reply:
I'm certain that I have seen "Mon" (and "MON" and "mon") used in reference to Ceph Monitors, but I share your intuition here and I have altered this to read "Ceph Monitor".